variables.tf

```
@@ -299,6 +299,18 @@ variable "ecs_task_memory" {
   default     = 512
 }
 
+variable "container_cpu" {
+  description = "The number of cpu units used by the atlantis container. If not specified ecs\_task\_cpu will be used"
```

we don't need to escape here and for some reason it makes TF unhappy!
```suggestion
  description = "The number of cpu units used by the atlantis container. If not specified ecs_task_cpu will be used"
```